<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Principal</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(11, 36, 71);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>591</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>22</width>
      <height>0</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Usuario">
     <attribute name="title">
      <string>Usuario</string>
     </attribute>
     <widget class="QTableWidget" name="tb_Usuario">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>671</width>
        <height>241</height>
       </rect>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>111</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>25</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>23</number>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nombre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Apellido</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>User</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Contraseña</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Fecha creación</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Apellido:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Fecha de Creación:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_Guardar_Usuario">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>20</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(165, 215, 232);</string>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_Editar_Usuario">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>70</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(165, 215, 232);</string>
      </property>
      <property name="text">
       <string>Editar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_Eliminar_Usuario">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>120</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(165, 215, 232);</string>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_Vaciar_Usuario">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>170</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(165, 215, 232);</string>
      </property>
      <property name="text">
       <string>Vaciar Campos</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_Usuario_Nombre">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>351</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_Usuario_Apellido">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>351</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_Usuario_User">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>351</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_Usuario_Password">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>210</y>
        <width>331</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QDateEdit" name="line_Usuario_Fecha">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>260</y>
        <width>271</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_Usuario_Id">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Rol</string>
     </attribute>
     <widget class="QPushButton" name="bt_Guardar_Rol">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>10</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(165, 215, 232);</string>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_Editar_Rol">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>60</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(165, 215, 232);</string>
      </property>
      <property name="text">
       <string>Editar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_Eliminar_Rol">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>110</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(165, 215, 232);</string>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_Vaciar_Rol">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>160</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(165, 215, 232);</string>
      </property>
      <property name="text">
       <string>Vaciar Campos</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Rol:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tb_Rol">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>641</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="autoScroll">
       <bool>true</bool>
      </property>
      <property name="autoScrollMargin">
       <number>18</number>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>317</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>40</number>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Rol</string>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="line_Rol">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>361</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_Rol_Id">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Opciones</string>
     </attribute>
     <widget class="QPushButton" name="bt_Guardar_Opcion">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>10</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(165, 215, 232);</string>
      </property>
      <property name="text">
       <string>Crear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_Editar_Opcion">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>60</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(165, 215, 232);</string>
      </property>
      <property name="text">
       <string>Editar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_Eliminar_Opcion">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>110</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(165, 215, 232);</string>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_Vaciar_Opcion">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>160</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(165, 215, 232);</string>
      </property>
      <property name="text">
       <string>Vaciar Campos</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tb_Opcion">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>641</width>
        <height>321</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="autoScrollMargin">
       <number>18</number>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>316</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>40</number>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Opcion</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Opcion:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_Opcion">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>361</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_Opcion_Id">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
